README.md

```diff
@@ -256,6 +383,17 @@ collections:
       audio_language: Français
 ```
 
+### Remove Items (Collection Attribute)
```

Comment:
Nice, thanks for adding this. A couple of thoughts on how to express it in the config:

- I'm not sure `remove_items` is intuitive enough for someone just picking up script. What about `sync_mode` and then the options are `append_only` (default) and `append_and_delete`? Feels like it would be easier for someone to understand without the README
- I think the best way is to have a setting for the entire config that can be overridden at the collection. What is we put `sync_mode` under `[plex]` with an `append_only` default and then exposed it as an option to override at the collection level?

Reply:
I like sync_mode, append_only (default), and append_and_delete those are much better names. Also having a global setting would be perfect Ill get working on both right away

Reply:
For `sync_mode`, how about `append` (default) and `overwrite` or `replace` as options? I agree with having a setting for the entire config that can be overridden for each collection. For example, you'd probably want an Academy Award winners or MCU collection to be append, but trending/popular/recent collection to be recreated from scratch.

Reply:
After pondering, I'd put the `sync_mode` outside of the `[plex]` section. Other name suggestions are `collection_mode`, `default_collection_mode`, or just `mode` with options `append` (default) or `overwrite`/`replace`/`refresh`.

Then within each collection, it can be overridden with `refresh: true` (or whatever word we picked).

Reply:
i like sync_mode over collection_mode mainly because we already have a collection_mode attribute lol
what about `add_only` and `add_and_remove` to match what the program actually prints out that it's doing

Reply:
Totally missed `collection_mode`. `sync_mode` works for me. I do still like `append`.
If not `overwrite`/`replace`/`refresh`, how about `sync_mode: full` for the override?

And since you mentioned the program printing what it's doing, I was also thinking about modifying the output from something like:
```
+++ Adding Avengers: Endgame to collection: Trending
Iron Man is already in collection: Trending
```
to
```
+ | Avengers: Endgame | Trending
= | Iron Man | Trending
- | Sharknado | Trending
```
(not that it has to be in this pull)

Reply:
im trying to start the prints with "| " and i also switched the item name with the collection name to make all the the the beginning have the same length so its easier to read ie

| Pixar Collection already contains: Brave
| Pixar Collection already contains: A Bug's Life
| Pixar Collection already contains: Cars 3
| --- Pixar Collection: Die Hard
| +++ Pixar Collection: Finding Nemo

but i like your idea of simplifying it how does this look 
| = | Pixar Collection | Brave
| = | Pixar Collection | A Bug's Life
| = | Pixar Collection | Cars 3
| - | Pixar Collection | Die Hard
| + | Pixar Collection | Finding Nemo

or maybe 
| = | Pixar Collection: Brave
| = | Pixar Collection: A Bug's Life
| = | Pixar Collection: Cars 3
| - | Pixar Collection: Die Hard
| + | Pixar Collection: Finding Nemo

we could also do (I like this one the best personally)
| Pixar Collection | = | Brave
| Pixar Collection | = | A Bug's Life
| Pixar Collection | = | Cars 3
| Pixar Collection | - | Die Hard
| Pixar Collection | + | Finding Nemo

but i really was trying to go for it to line up when printing to make it easier to read which means the movie name at the end

Reply:
i agree i do think we should change the layout for adding as detailed above but as for the names i think we should go with either `append` or `append_only` for the first and for the second i just dont feel like its actually overwriting/replacing/refreshing anything so those mode names don't seem intuitive to me. `full` is better but i just don't feels it convays to the user that it means to remove items from the collection which is why i like the `append_and_remove`.

Another option could be `sync_mode: sync` vs `sync_mode: append`

Reply:
ok so I added the global attribute under plex in the config and set them as `sync_mode: sync` and `sync_mode: append` ill change this if needed but it was better then it was

Reply:
i also updated it to the 3rd printout option that i detailed above it looks really clean